ucum-java8/src/main/java/systems/uom/ucum/format/UCUMFormat.java

```diff
@@ -286,12 +287,26 @@ public Appendable format(Unit<?> unknownUnit, Appendable appendable) throws IOEx
 	    symbol = temp;
 	} else if (unit.getSymbol() != null) {
 	    symbol = unit.getSymbol();
-	} else {
+	} else if (unknownUnit instanceof TransformedUnit) {
+            final StringBuilder temp = new StringBuilder();
+
+            Unit<?> parentUnit = ((TransformedUnit) unit).getParentUnit();
+            UnitConverter converter;
```

Comment:
I don't know why you're declaring your variables way above where you initialize them. If you like that style, fine. But people have no way of knowing if they are getting initialized correctly or not.

Now if you were to have made them `final`, then the compiler could tell you if they were initialized or not. (I don't understand why people hate using `final` so much. It's so useful and has prevented bugs for me.)